src/views/profile.hbs

```diff
@@ -0,0 +1,106 @@
+<div id="modal-details" class="modal">
+  <div class="modal-background">
+    {{> profile_myDetails }}
+  </div>
+</div>
+
+
+
+<nav id="navigation">
+  <a href="/allusers" class="" id="">ALL USERS</a>
+  <a href="" class="" id="">MY PROFILE</a>
+</nav>
+
+<header>
+  <h1>My Profile</h1>
+  <div>
+    <a href="/myprofile/:github_id" class="" id="">View Profile</a>
+    <a href="/myprofile/:github_id" class="" id="">Edit Profile</a>
+  </div>
+</header>
+
+<article>
+  <section>
+
+
+    <header>
+      <h1>
+        MY DETAILS
```

Comment:
You can use CSS to transform text to uppercase, which may be more ideal than hardcoding uppercase text.